engine/dlib/src/dmsdk/dlib/log.h

```diff
@@ -278,38 +288,9 @@ void dmLogSetLevel(LogSeverity severity);
 
 namespace dmLog
 {
-
-/*# register dmLog listener.
- *
- * Registers a dmLog listener.
- * This listener receive logs even in release bundle.
- *
- * @note deprecated. Use dmLogRegisterListener instead.
- * @name dmLog::RegisterLogListener
- * @param listener [type:FLogListener]
- */
-void RegisterLogListener(FLogListener listener);
-
-/*# unregister dmLog listener.
- *
- * Unregisters a dmLog listener.
- *
- * @note deprecated. Use dmLogUnregisterListener instead.
- * @name dmLog::UnregisterLogListener
- * @param [type:FLogListener] listener
- */
-void UnregisterLogListener(FLogListener listener);
-
-/*# set log system severity level.
- *
- * set log system severity level.
- *
- * @note deprecated. Use dmLogSetLevel instead.
- * @name dmLog::Setlevel
- * @param [type:dmLog::Severity] severity
- */
-void Setlevel(LogSeverity severity);
-
+    void RegisterLogListener(FLogListener listener);
+    void UnregisterLogListener(FLogListener listener);
+    void Setlevel(LogSeverity severity);
```

Comment:
I've deprecated the dmLog namespace again, since it didn't work well with C code.